### ISSUE TITLE
Question about generalizable

### ISSUE THREAD

**Issue by SuzyZhao0417** [dcharatan/pixelsplat#44]
Hello! Thank you for your excellent work! Regarding generalization, I have some questions. Could you please clarify whether your model achieves scale generalization within the same scene, or if it utilizes pre-trained models to achieve generalization on unseen scenes? Looking for your reply! Best wishes!

**Comment by dcharatan** [comment]
Can you explain what you mean by achieving scale generalization within the same scene? You can overfit to a single scene (see #40), but during overfitting of any NeRF-like model, scale ambiguity isn't a concern. The model is designed to not require metric-scale camera poses during generalization to new scenes.
